### ISSUE TITLE
unable to build goproxy

### ISSUE THREAD

**Issue by spykermj** [snail007/goproxy#413]
## Expected Behavior

I would expect an open source project to provide a documented build process

## Current Behavior

The only files in the repository that mention building (in Enlish anyway) pull a pre-built image from a web site instead of using golang to build the project

## Steps to Reproduce

1. place the goproxy code into your go root/src directory as "proxy"
1. run `go get -d proxy` from the go root src directory
1. cd proxy
1. go build main.go

## Context (Environment)

1. proxy version is : v? trying to build from the current master branch
1. full command is :? Details are given in the "steps to reproduce"
1. system is : CentOS
1. full log is: ? 

# command-line-arguments
go build main.go
./main.go:15:9: undefined: initConfig
./main.go:19:9: undefined: service

## Detailed Description

I propose that a repeatable build process should be provided

**Comment by shinvdu** [comment]
I have the same issue too.
--------------------
silas@myhost goproxy-10.2  % go build main.go
main.go:8:2: cannot find package "proxy/services" in any of:
	/usr/local/go/src/proxy/services (from $GOROOT)
	/home/silas/go/src/proxy/services (from $GOPATH)
silas@silas-ThinkPad-L460 goproxy-10.2  % 
--------

我想说是， 即然， 都开源了， 为什么不能让他好好build成功呢，让大家可以在上面修改， 加更多的功能， 才是开源的本质。

最后， 很感谢作者的开源软件。

**Comment by TonyRL** [comment]
@spykermj There's one critical part missing if you compare the README with README_ZH.

https://github.com/snail007/goproxy/blob/master/README_ZH.md#源代码申明

A simple Google Translate
> Source code declaration
The author of this project found that a large number of developers based on this project for secondary development or use a large amount of the core code of this project without following the GPLv3 agreement, which seriously violates the original intention of this project to use the GPLv3 open source agreement. In view of this situation, this project adopts the source The code delayed release strategy, to a certain extent, curbs these disrespect for open source and disrespect for the work of others.
This project will continue to update and iterate, continue to release full-platform binary programs, and provide powerful and convenient agent tools for everyone.

TL;DR version
This repo contains outdated source but updated binary, docs, etc.

**Comment by spykermj** [comment]
OK, my Chinese language skills aren't so good. It sounds like this is no longer an open source project.
